pyproject.toml

```diff
@@ -31,9 +31,12 @@ dev = [
     "types-mock",
     "atomicwrites",
     "typed-ast",
+    "copier",
+    "myst-parser",
     "pipdeptree",
     "pre-commit",
     "pydata-sphinx-theme>=0.12",
+    "pyright",
```

Comment:
Either pyright or mypy, not both. Thought pyright is complaining a lot about the codebase so if we keep it then we'll have to fix that in a different PR.